numba/unicode.py

```diff
@@ -540,6 +540,93 @@ def find_impl(a, b):
         return find_impl
 
 
+@register_jitable
+def _check_type(ty, name, accepted=(types.NoneType,)):
```

Comment:
This function probably ought to be called something associated with its use, like `unicode_rfind_check_type` also, why isn't it just an inner function in the overload, the scope then prevents naming issues?

Reply:
I will make the change.